test/ads_integration.js

```diff
@@ -40,151 +38,153 @@ filterDescribe('Ads', supportsImaWithoutPolyfills, () => {
       'unviewed_position_start=1&' +
       'cust_params=deployment%3Ddevsite%26sample_ct%3Dlinear&correlator=';
 
-  beforeAll(async () => {
-    await new Promise((resolve, reject) => {
-      imaScript = /** @type {!HTMLScriptElement} */(
-        document.createElement('script'));
-      imaScript.defer = false;
-      imaScript['async'] = false;
-      imaScript.onload = resolve;
-      imaScript.onerror = reject;
-      imaScript.setAttribute('src',
-          'https://imasdk.googleapis.com/js/sdkloader/ima3.js');
-      document.head.appendChild(imaScript);
+  if (!shaka.util.Platform.isTizen3()) {
```

Comment:
I think the version from the previous commit (filterDescribe) makes more sense for me.  But your condition was inverted:

```js
const supportsImaWithoutPolyfills = () => shaka.util.Platform.isTizen3();
```

Should have been:

```js
const supportsImaWithoutPolyfills = () => !shaka.util.Platform.isTizen3();
```

Reply:
It’s not possible! Load IMA script breaks Tizen 3, so we need avoid load to the script. filterDescribe run before* and after* codes

Reply:
That would make a great code comment!

Reply:
done!